### PR TITLE
Geocode every location entries in new case

### LINKED ISSUE #1231: Geocode travel history as well
**Is your feature request related to a problem? Please describe.**
trel history locations are not geocoded right now, one cannot set the location.query in them and have it geocoded.

**Describe the solution you'd like**
We should just geocode them as well.

### PR DESCRIPTION
**Pull request by attwad**
So far we were only looking at `case.location`, now we also look at `case.travelHistory.travel.location`.

Fixes #1231

### CODE REVIEW

File: data-serving/data-service/test/controllers/case.test.ts [hunk 1]
```diff
@@ -492,6 +493,47 @@ describe('POST', () => {
...
+            .expect(200);
+        await request(app)
+            .post('/api/cases')
+            .send({
+                ...caseMustGeocode,
+                ...curatorMetadata,
+            })
+            .expect(201)
+            .expect('Content-Type', /json/);
+        expect(await Case.collection.countDocuments()).toEqual(1);
```
Comment (allysonjp715) [inline data-serving/data-service/test/controllers/case.test.ts:524]: Can you check that the created case has the right geocoded fields?

**Review by allysonjp715 — APPROVED** [review]

File: data-serving/data-service/test/controllers/case.test.ts (hunk 1 above)
Comment (attwad) [reply data-serving/data-service/test/controllers/case.test.ts]: yup, done.

**Comment by Mougk** [comment]
@attwad do we need to adjust our parsers for this or should geo-coding on travel history work fine as is?

**Comment by attwad** [comment]
No parser was providing that info until now, Felix is adding the first one right now so nothing needs to be done retroactively.

**Pull request merged**